bower.json

```diff
@@ -20,16 +20,15 @@
   "private": true,
   "license": "http://polymer.github.io/LICENSE.txt",
   "homepage": "https://github.com/PolymerElements/paper-header-panel",
-  "ignore": [],
```

Comment:
Yeaaaaaah, we seem to get bower warnings like `"invalid-meta iron-component-page is missing "ignore" entry in bower.json"` if we don't have them, so please add it back :)